src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -272,16 +273,30 @@ def add_image_segmentation(nwbfile: NWBFile, metadata: dict) -> NWBFile:
 
 
 def add_two_photon_series(
-    imaging, nwbfile, metadata, buffer_size=10, use_times=False, two_photon_series_index: int = 0
+    imaging,
+    nwbfile,
+    metadata,
+    two_photon_series_index: int = 0,
+    use_times=False,
+    buffer_size: Optional[int] = None,
+    iterator_type: Optional[str] = None,
```

Comment:
```suggestion
    iterator_type: str = "v2",
```
I'd use the new one by default, which can be modified if someone insists to use the other one.